### DISCUSSION TITLE
Grey Screen

### DISCUSSION THREAD

**Discussion by JustAnotherView** [The-Wineskin-Project/WineskinServer#428] [Q&A]
Having a problem where the game launches into a grey screen and I can't figure out what the issue is. 
[LastRunWine.log](https://github.com/Gcenx/WineskinServer/files/11995531/LastRunWine.log)

**Comment by Gcenx** [comment]
According to the provided log your system had an Intel iGPU, that’s not viable for playing modern games on macOS.

**Reply by JustAnotherView** [reply]
I have no problem running heavier games like Guild Wars 2 or even pc ports of console games though.
